### DISCUSSION TITLE
End of support date of Symfony 2.8

### DISCUSSION THREAD

**Discussion by Maximilien92330** [symfony/symfony#47322] [Q&A] [answered]
Hello,
I work for MEGA International.
I am an IT Consultant and my current client wants to update their technology repository information.
It features the Symfony 2.8 technology.
We would like to know the end of support date as well as the end of extended support date for Symfony 2.8.
Could you please tell me these two dates?
The most important information being the end of support date.

I thank you in advance.
Cordially,
Maximilien Buey

**Comment by stof** [comment, marked as answer]
See https://symfony.com/releases/2.8
